source/extensions/tracers/lightstep/config.cc

```diff
@@ -15,19 +15,29 @@ namespace Extensions {
 namespace Tracers {
 namespace Lightstep {
 
-Tracing::HttpTracerPtr LightstepTracerFactory::createHttpTracer(const Json::Object& json_config,
-                                                                Server::Instance& server) {
+Tracing::HttpTracerPtr
+LightstepTracerFactory::createHttpTracer(const envoy::config::trace::v2::Tracing& configuration,
+                                         Server::Instance& server) {
+  ProtobufTypes::MessagePtr config_ptr =
```

Comment:
Can you factor out this to Server::Configuration and let the interface have `createEmptyProto` as well, just like other extensions? This will make extension interface stable and easier for #4475.